/vampire/evologics_ros_node:
  ros__parameters:
    goby:
      max_frame_bytes: 1000
      mac_slot_time: 10
      valid_depth: -1

      dynamic_buffer:
        #select the subbuffer_id(s) to load to the dynamic buffer
        messages: ["vampire_state"]

        #buffer configuration (https://goby.software/3.0/md_doc101_acomms-queue.html)
        vampire_state:
          ack: false
          blackout_time: 0
          max_queue: 1
          newest_first: true 
          ttl: 1800
          value_base: 1

